src/components/App/App.tsx

```diff
@@ -4,6 +4,7 @@ import Navbar from '../Navbar/Navbar';
 import Main from '../Main/Main';
 import BreadList from '../BreadList/BreadList';
 import BreadDetail from '../BreadDetail/BreadDetail';
+import Error from "../Error/Error";
```

Comment:
Ok and this is the error page handling stuff being imported to handle pages that shouldn't have routing to them.